Rocket_Stabilization/RollPitchYaw/RollPitchYaw-udb5.mcp

```diff
@@ -317,6 +317,6 @@ transport=0
 format=0
 [CUSTOM_BUILD]
 Pre-Build=cscript "$(ProjectDir)..\libUDB\fixDeps.vbs" "$(ProjectDir)options.h" "$(ProjectDir)..\libUDB\fixDeps.h"cscript "$(ProjectDir)..\libUDB\fixDeps.vbs" "$(ProjectDir)options.h" "$(ProjectDir)..\libUDB\fixDeps.h"
-Pre-BuildEnabled=1
+Pre-BuildEnabled=0
```

Comment:
@wjpremerlani I think this change might fix the issue that you're experiencing, it doesn't seem to change anything for me, but seems to be related.